Client/mods/deathmatch/logic/CClientVectorGraphic.cpp

```diff
@@ -0,0 +1,82 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *  LICENSE:     See LICENSE in the top level directory
+ *
+ *  Multi Theft Auto is available from http://www.multitheftauto.com/
+ *
+ *****************************************************************************/
+
+#include "StdInc.h"
+#include "CClientVectorGraphic.h"
+#include <svgdocument.h>
+
+CClientVectorGraphic::CClientVectorGraphic(CClientManager* pManager, ElementID ID, std::unique_ptr<CVectorGraphicItem> pVectorGraphicItem)
+    : ClassInit(this), CClientTexture(pManager, ID, pVectorGraphicItem.get())
+{
+    SetTypeName("svg");
+
+    m_pDocument = nullptr;
+    m_pResource = nullptr;
+    m_pManager = pManager;
+
+    m_pVectorGraphicItem = std::unique_ptr<CVectorGraphicItem>(std::move(pVectorGraphicItem));
```

Comment:
Just do `m_pVectorGraphicItem = std::move(pVectorGraphicItem)`. No need for a new `unique_ptr`